java/arcs/sdk/wasm/WasmJsInterop.kt

```diff
@@ -250,5 +250,106 @@ fun log(msg: String) {
     flush()
 }
 
+/** Get a combined handle to call events on. */
```

Comment:
This could be a bit more detailed, describing what it does and how to use it correctly.